packages/bus-rdf-resolve-quad-pattern/lib/ActorRdfResolveQuadPattern.ts

```diff
@@ -14,6 +14,11 @@ export const KEY_CONTEXT_SOURCES = '@comunica/bus-rdf-resolve-quad-pattern:sourc
  * @value {IDataSource} A source.
  */
 export const KEY_CONTEXT_SOURCE = '@comunica/bus-rdf-resolve-quad-pattern:source';
+/**
+ * @type {string} Context entry for the authentication for a source.
+ * @value {string} "username:password"-pair.
+ */
+export const KEY_CONTEXT_AUTH = '@comunica/bus-http:auth';
```

Comment:
Could you put this in the http bus instead? There should already be some context keys in its main abstract actor AFAIK.